scripts/build.sh

```diff
@@ -162,6 +166,9 @@ esac
 
 echo "Building OpenSearch for $PLATFORM-$DISTRIBUTION-$ARCHITECTURE"
 
+
+wget -q https://raw.githubusercontent.com/wazuh/wazuh/$BRANCH/extensions/elasticsearch/7.x/wazuh-template.json -O distribution/src/config/wazuh-template.json
```

Comment:
Some suggestions:

- Use cURL.
- Add error handling. What happens if the download fails?
- To make error handling and testing easier, consider creating a function.

Reply:
I just pushed a commit that I believe answers all these points. Thanks for the heads up.